### ISSUE TITLE
GRPO Training experience for HunyuanVideo

### ISSUE THREAD

**Issue by k15201363625** [XueZeyue/DanceGRPO#34]
Thanks to the author for open source and actively responding to issues!
In reproduce of HunyuanVideo T2V GRPO training, I'm using 8 GPUs and have increased the batch size to ensure the total batch size isn’t too small (4 batch_size * 2 acc * 8 cards) and keep the provided parameter settings (num_generation=24, bestofn=8).

For four experiments: training on the original dataset(50000 samples) and the MovieGen dataset(1000 samples), with and without LoRA (rank 256, alpha 128). Across all these attempts, even after more than 100 training steps, the VQ reward increase is not seen.

Are there any suggestion for adjusting the hyperparameters? 😭
For example:
Does a larger total batch size is good for stability of training?
What is the suitable prompt dataset scale?  Do prompts need to be refined by VLM?
Can LoRA cause the training crash more easily?
Are the settings of height, weight, temporal length important? (Previously, during reward tuning with WAN, the original resolution was needed.)
Why transformer forward with parameter "guidance=torch.tensor([6018.0]...)" ?
Is it better to turn use_same_noise on or off for GRPO training?

Thanks so much!
(The current training curves shown below, which reward values are unscaled.)

<img width="1089" height="569" alt="Image" src="https://github.com/user-attachments/assets/c1a4b533-b239-476a-b8d9-d5ca012259b7" />
